torchvision/models/shufflenetv2.py

```diff
@@ -68,10 +74,11 @@ def __init__(self, inp, oup, stride):
         )
 
     @staticmethod
-    def depthwise_conv(i, o, kernel_size, stride=1, padding=0, bias=False):
+    def depthwise_conv(i: int, o: int, kernel_size: int, stride: int = 1, padding: int = 0,
+                       bias: bool = False) -> nn.Conv2d:
```

Comment:
Could you format this with `black` to make it more legible?

Reply:
Fixed!